summingbird-scalding/src/main/scala/com/twitter/summingbird/scalding/batch/BatchedStore.scala

```diff
@@ -93,14 +105,14 @@ trait BatchedStore[K, V] extends scalding.Store[K, V] { self =>
 
   protected def sumByBatches[K1, V: Semigroup](ins: TypedPipe[(Timestamp, (K1, V))],
     capturedBatcher: Batcher,
-    commutativity: Commutativity): TypedPipe[((K1, BatchID), (Timestamp, V))] = {
+    commutativity: Commutativity): TypedPipe[(LTuple2[K1, BatchID], (Timestamp, V))] = {
     implicit val timeValueSemigroup: Semigroup[(Timestamp, V)] =
       IteratorSums.optimizedPairSemigroup[Timestamp, V](1000)
 
     val inits = ins.map {
       case (t, (k, v)) =>
         val batch = capturedBatcher.batchOf(t)
-        ((k, batch), (t, v))
+        (LTuple2(k, batch), (t, v))
```

Comment:
this is going to make it so that each key, batch pair is unique in the map, so the values will never be reduced, right? Why not just skip mapside aggregation entirely?